test/operator/operator_test.py

```diff
@@ -213,7 +191,9 @@ def test_linear_Op():
     x = np.random.rand(3)
     out = np.random.rand(3)
 
-    Aop = MultiplyOp(A)
+    r3 = odl.Rn(3)
+
+    Aop = MatVecOperator(r3, r3, A)
```

Comment:
I implemented this now in e98cdc06f66ff31eadaaa7c09dc1a3de7820eb50. I suggest you rebase and, if you want, remove the `rn`s from the class initializer. If not, the matrix must come first now.